config/initializers/locations.rb

```diff
@@ -1 +1 @@
-Locations.path = Rails.root.join('lib', 'locations', 'locations.json')
+Locations.path = 'http://0.0.0.0:8080/locations.json'
```

Comment:
It's confusing. I have to know that `Locations::Repository` is using open-uri and therefore I can provide a URL and it will work.